accepted/calc-functions.md

```diff
@@ -620,6 +625,36 @@ To evaluate a `SumExpresssion` or a `ProductExpression` as a calculation value:
 
 * Return `left`.
 
+### `SpaceListExpression`
+
+To evaluate a `SpaceListExpresssion` as a calculation value:
+
+* Let `elements` be the results of evaluating each element as a calculation
+  value.
+
+* If `elements` has two adjacent elements that aren't unquoted strings, throw an
```

Comment:
Would this accept `calc(a b)` and if so, is that OK?

Reply:
Yes; I've updated the non-normative note to mention this.